ui/app/AppLayouts/Chat/views/communities/CommunityNewPermissionView.qml

```diff
@@ -496,7 +536,7 @@ StatusScrollView {
             ColumnLayout {
                 Layout.fillWidth: true
                 StatusBaseText {
-                    text: qsTr("Private")
+                    text: qsTr("Hide permission")
```

Comment:
The description also needs to be updated to: "Make this permission hidden from members
who don’t meet it’s requirements"